app/helpers/search_helper.rb

```diff
@@ -8,4 +8,10 @@ def trim_highlights(result)
 
     result['highlight'].reject { |h| displayed_fields.include? h['matchedField'] }
   end
+
+  def view_online(result)
+    return unless result['sourceLink'].present?
+
+    link_to 'View online', result['sourceLink'], class: 'button button-primary green'
```

Comment:
I don't think this has been introduced as a change for this PR, but I noticed that `green` is only applied on hover or if I've visited the link. Is that intended? I'm not asking for a change, just curious if we are intentionally having the button "black" for unclicked and "green" for clicked (which seems contrary to what this class applies hence my question).

Reply:
Yes, the intended behavior is that the button's background color is black normally, then green on hover and when visited. The 'green' class comes from our design system; it adds the `a:hover { background-color: green}` rule to our link-buttons instead of the standard `a:hover { background-color: blue }`.